iquip/apps/builder.py

```diff
@@ -403,8 +403,8 @@ def value(self) -> Dict[str, Any]:
         
         Returns the dictionary of the selected scannable arguments.
         """
-        selected = self.state["selected"]
-        return self.state[selected]
+        selectedWidget = self.stackedWidget.currentWidget()
```

Comment:
How about `selectedScanWidget` as you referred this as `scanWidget` in `__init__()`?